recipes/mlapack/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.8.1" %}
+{% set revision = "6b8193bac747fc21362253df1e7c7dd50bcb5ff1" %}
+
+package:
+  name: mlapack 
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/nakatamaho/mplapack
```

Comment:
Is that because they are using git submoldules? Preferred is a tarball. You could ask upstream to do a release as well.